setup.cfg

```diff
@@ -1,26 +1,32 @@
-[bumpversion]
-current_version = 0.1.0
-commit = True
-tag = True
+[metadata]
+name = git_wrapper
+author = Jason Joyce
+author-email = jjoyce@redhat.com
+summary = Python commodity wrapper around GitPython
+description-file =
+    README.rst
+    HISTORY.rst
+home-page = https://github.com/release-depot/git_wrapper
+classifiers =
+    Development Status :: 2 - Pre-Alpha
+    Intended Audience :: Developers
+    License :: OSI Approved :: MIT License
+    Natural Language :: English
+    Programming Language :: Python :: 3
+    Programming Language :: Python :: 3.5
+    Programming Language :: Python :: 3.6
+keywords =
+    git_wrapper
 
-[bumpversion:file:setup.py]
-search = version='{current_version}'
-replace = version='{new_version}'
-
-[bumpversion:file:git_wrapper/__init__.py]
-search = __version__ = '{current_version}'
-replace = __version__ = '{new_version}'
-
-[bdist_wheel]
-universal = 1
-
-[flake8]
-exclude = docs
+[files]
+packages =
+    git_wrapper
 
 [aliases]
-# Define setup.py command aliases here
 test = pytest
 
 [tool:pytest]
-collect_ignore = ['setup.py']
+addopts = tests
```

Comment:
Should this be test instead of tests?

Reply:
Nope, this is the tests directory (if I got your question right). By default, it looks inside the main package directory, but since it's outside, we need this  section to indicate that.

Reply:
That makes sense.

Reply:
This is handled in setup.py (the line with setup_requires=['pbr'])